vault/testing.go

```diff
@@ -2177,19 +2177,20 @@ func (tc *TestCluster) initCores(t testing.T, opts *TestClusterOptions, addAudit
 		}
 
 		// Let them come fully up to standby
-		time.Sleep(2 * time.Second)
-
-		// Ensure cluster connection info is populated.
-		// Other cores should not come up as leaders.
-		for i := 1; i < numCores; i++ {
-			isLeader, _, _, err := tc.Cores[i].Core.Leader()
-			if err != nil {
-				t.Fatal(err)
-			}
-			if isLeader {
-				t.Fatalf("core[%d] should not be leader", i)
+		assert.Eventually(t, func() bool {
```

Comment:
It's cool that there's a library function for this in testify, but I kind of prefer corehelpers.RetryUntil - that way we get to see the error we were getting when we timed out.